src/main.go

```diff
@@ -149,7 +151,12 @@ func main() {
 		renderer:     renderer,
 		consoleTitle: title,
 	}
-	engine.render()
+
+	if *args.Debug {
+		engine.debug()
+	} else {
```

Comment:
change this for:

```golang
if *args.Debug {
    engine.debug()
    return
}
engine.render()
```

I have a strong opinion against using `if else` when an early return is more elegant.